docker-compose.katana.yaml

```diff
@@ -0,0 +1,48 @@
+version: "3.2"
+
+services:
+  katana:
+    image: ghcr.io/dojoengine/dojo:latest
+    command: ["katana", "--allow-zero-max-fee"]
```

Comment:
We usually run katana with these

```
--validate-max-steps 16777216 --invoke-max-steps 16777216 --gas-price 0
```

Reply:
Gotcha let me update this with those values 🙂